addons/l10n_be_pos_restaurant/data/template/account.tax-be.csv

```diff
@@ -0,0 +1,2 @@
+id,name,description,amount,amount_type,type_tax_use
```

Comment:
This tax is not going to appear in the tax report if it doesn't have any tags.
It can be changed in stable, but keep that in mind